src/Commands/SyncCommand.php

```diff
@@ -38,7 +38,7 @@ protected function syncScheduledTasksWithDatabase(): self
         $monitoredScheduledTasks = ScheduledTasks::createForSchedule()
             ->uniqueTasks()
             ->map(function (Task $task) {
-                return MonitoredScheduledTask::updateOrCreate(
+                return app(MonitoredScheduledTask::class)::updateOrCreate(
```

Comment:
To me, using `::` twice in a statement hurts readability.

Maybe you could add a trait `UsesScheduleMonitoringModels` that has a function `getMonitoredScheduleTaskModel` and use that wherever you need to use that model.

Reply:
@freekmurze Sounds good. Great suggestion. 👍 I'll get that implemented.

Reply:
@freekmurze I've gone ahead and changed out the references to the classes to now use a trait.  I've also added a test that checks that the trait methods return instances of the expected models.